poetry/masonry/utils/package_include.py

```diff
@@ -44,10 +44,10 @@ def check_elements(self):  # type: () -> PackageInclude
             # Probably glob
             self._is_package = True
 
-            # The __init__.py file should be first
+            # Packages no longer need an __init__.py in python3
             root = self._elements[0]
-            if root.name != "__init__.py":
```

Comment:
I think we can change this line to check if there is at least one Python file, regardless of whether it's a `__init__.py` file or not. If there is no Python file, it's not a package and we should raise the exception.

Reply:
Thanks, I've also added it now in commit 1d0e99d.